node/hello-world/index.js

```diff
@@ -1,4 +1,4 @@
-// A version of this script is available for node v4 in index.v4.js
+// A version of this script is available for node v6 in index.v6.js
```

Comment:
Can you please add a comment here for the node.js version for which this sample will work?